examples/NightSkyAnimation.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2015-2017 WorldWind Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ *  Illustrates how to animate the passage of time with a day/night cycle on the surface of the globe,
+ *  as well as the starry sky above the Earth (with realistic star positions).
+ */
+requirejs([
+        './WorldWindShim',
+        './LayerManager'
+    ],
+    function (WorldWind,
+              LayerManager) {
+        'use strict';
+
+        // Tell WorldWind to log only warnings and errors.
+        WorldWind.Logger.setLoggingLevel(WorldWind.Logger.LEVEL_WARNING);
+
+        // Create the WorldWindow.
+        var wwd = new WorldWind.WorldWindow('canvasOne');
+
+        // Create imagery layers.
+        var BMNGOneImageLayer = new WorldWind.BMNGOneImageLayer();
+        var BMNGLayer = new WorldWind.BMNGLayer();
+        wwd.addLayer(BMNGOneImageLayer);
+        wwd.addLayer(BMNGLayer);
+
+        // Use the StarField layer to show stars and the Sun around the globe, and the Atmosphere layer to display
+        // the atmosphere effect and the night side of the Earth.
+        // Note that the StarField layer requires a dark canvas background color.
+        // The StarField layer should be added before the Atmosphere layer.
+        var starFieldLayer = new WorldWind.StarFieldLayer();
+        var atmosphereLayer = new WorldWind.AtmosphereLayer();
+        wwd.addLayer(starFieldLayer);
+        wwd.addLayer(atmosphereLayer);
+
+        // Set a date property for the StarField and Atmosphere layers to the current date and time.
+        // This enables the Atmosphere layer to show a night side (and dusk/dawn effects in Earth's terminator).
+        // The StarField layer positions its stars according to this date.
+        var now = new Date();
+        starFieldLayer.time = now;
+        atmosphereLayer.time = now;
+
+        // Animate the starry sky as well as the globe's day/night cycle.
```

Comment:
Comment without code. Needs to be moved or removed. Probably best above line 82.